bin/run_bnb.sh

```diff
@@ -0,0 +1,6 @@
+ #! /bin/sh
+export DRY_RUN=1
```

Comment:
Since the docker-compose has a local Kafka, I think the default mode should not be DRY_RUN